classes/Api.js

```diff
@@ -63,6 +66,26 @@ module.exports = function () {
         return scanner.execute(scanRequest);
     };
 
+    _this.finishMerge = function (req) {
+        var dateString = dateFormat(new Date(), 'yyyy-mm-dd HH.MM.ss');
+        var outputFilepath = Config.OutputDirectory + 'scan_' + dateString;
+        
+        var options = {
+            target: outputFilepath,
+            pages: req.pages,
```

Comment:
Make sure that all that paths of all pages lead to files inside ``Config.OutputDirectory`` or it might be possible to access files outside that path (e.g. readable system configuration).